css-four-column-layout/styles.css

```diff
@@ -0,0 +1,50 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Barlow, sans-serif;
+  font-size: 0.7rem;
+}
+
+h1,
+h4 {
+  font-family: Barlow, sans-serif;
+}
+
+.how-it-works {
+  background-color: #f7f8fa;
+  text-align: center;
+}
+
+.max-width {
+  max-width: 700px;
+  margin: 0 auto;
+}
```

Comment:
This should be your `container` class

Reply:
![Screen Shot 2022-08-30 at 10 24 00 AM](https://user-images.githubusercontent.com/105886708/187503343-824d0f6d-7ef5-42b3-81a9-86e443a6e5e8.png)
So I would need to add a container class with the standard measurements for all elements with container. And add the .max-width-shrink with my desired measurements for this particular container?